api/tests/views_tests.py

```diff
@@ -12,33 +12,35 @@
 
 
 @pytest.fixture
-def free_api_client(db) -> APIClient:
+def free_user(db) -> User:
+    return make_free_test_user()
+
+
+@pytest.fixture
+def free_api_client(free_user: User) -> APIClient:
```

Comment:
question (non-blocking): so when multiple pytest fixtures are passed as parameters in test functions, the later fixtures receive the preceding fixture's `return` value as an argument?

Reply:
yes, or `yeild whatever` if you use that to allow post-test teardown